src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -62,10 +67,10 @@
 # if no specific metadata is provided for a tool, the default values are used.
 TASK_META = {'gtdb_tk': {'chunk_size': 1000, 'exe_time': 65, 'tasks_per_node': 4, 'threads_per_tool_run': 32},
              'eggnog': {'chunk_size': 100, 'exe_time': 15, 'node_time_limit': 0.5},  # Memory intensive tool - reserve more nodes with less node reservation time
+             'busco': {'chunk_size': 50, 'exe_time': 90, 'node_time_limit': 1.5},  # 1.5 minutes per genome with a single task per node on the user's drive. TODO: Aim to test multi-threading per node along with scratch execution, and adjust `tasks_per_node` accordingly.
```

Comment:
Wow this is expensive. For 10k genomes that's 250 hours, so scaling this to GTDB sized genome counts seems infeasible. Hopefully your TODOs pan out.